### PR TITLE
Add in scraping of options info (IV Rank/ Hist IV)

### PR DESCRIPTION
**Pull request by jmaslek**
Trying to address #368 , this PR scrapes Barnhart.com for the options info.

For some reason, requests does not work, so I added selenium to the requirements/poetry.

Selenium is a little more confusing, but I added the link where the driver can be downloaded for either chrome or firefox.

@aia I tried adding a unittest to make sure the issue with cvxpy doesn't happen, but that just messes with the workflows so I will eliminate it on the next commit.

### CODE REVIEW

File: gamestonk_terminal/options/op_scrape_view.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        default=None,
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+        if ns_parser.browser:
+            browser = ns_parser.browser
+        else:
```
Comment (DidierRLopes) [inline gamestonk_terminal/options/op_scrape_view.py:59]: Instead of this, you can add web as default of ns_parsers browser argument.

**Review by DidierRLopes — APPROVED** [review]
This looks good to me.

My only concern is: Is it it worth the effort given the information provided?

Regardless, I think that we could merge it, and if we found a better source for this info, we could probably update to use such source. Right?

**Comment by jmaslek** [comment]
I'll leave that to you.  Selenium is a good package but the fact that the outside driver is needed may make it complicated.

**Comment by DidierRLopes** [comment]
> I'll leave that to you. Selenium is a good package but the fact that the outside driver is needed may make it complicated.

I'm happy with it :) Let's go for it! If we stumble across something better in the future we'll think about it, but this is good! Cheers :) 

Feel free to merge. Before, can you just update:
* gamestonk_terminal/README with this new feature
* ROADMAP.md with this PR feature

**Pull request merged**